app/views/rooms/photos.html.erb

```diff
@@ -20,15 +20,15 @@
             <% end %>
             <div class="card-group mx-5">
                 <div class="card mx-4" style="width: 18rem;">
-                    <%= image_tag "house1.jpeg", class: "d-block mx-auto img-fluid w-100" %>
+                    <%= image_tag @room.images.first.image, class: "d-block mx-auto img-fluid w-100" %>
                     <%= link_to raw("<i class='fa fa-trash text-danger'></i>"), room_image_path(@room), method: :delete, data: { confirm: "You sure?" } %>
                 </div>
                 <div class="card mx-4" style="width: 18rem;">
-                    <%= image_tag "house2.jpg", class: "d-block mx-auto img-fluid w-100" %>
+                    <%= image_tag @room.images.second.image, class: "d-block mx-auto img-fluid w-100" %>
                     <%= link_to raw("<i class='fa fa-trash text-danger'></i>"), room_image_path(@room), method: :delete, data: { confirm: "You sure?" } %>
                 </div>
                 <div class="card mx-4" style="width: 18rem;">
-                    <%= image_tag "house3.jpg", class: "d-block mx-auto img-fluid w-100" %>
+                    <%= image_tag @room.images.third.image, class: "d-block mx-auto img-fluid w-100" %>
                     <%= link_to raw("<i class='fa fa-trash text-danger'></i>"), room_image_path(@room), method: :delete, data: { confirm: "You sure?" } %>
                 <div class="card-body">
```

Comment:
these lines of code are causing an issue, please fix them
![image](https://user-images.githubusercontent.com/73863408/133208987-4355bf03-86b4-4f27-b90d-26abdfcd2449.png)

_HINT:_
inside the rooms_controller you can try adding an `@image` variable that will receive the `@room.images` <- this code will fetch all images for the specific listing

using the `@image` variable you can then use `.each` to display and loop through each images